content/docs/services/cdn-route.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: CDN service
 name: "cdn-route"
-description: "Custom routes with HTTPS certificates using Amazon CloudFront"
+description: "Custom domains, CDN caching, and TLS certificates using Amazon CloudFront"
```

Comment:
Actually, just `Custom domains, CDN caching, and TLS certificates` would be fine here - we can leave the CDN details to the documentation